res/css/views/rooms/_RoomTile.pcss

```
@@ -55,13 +55,17 @@ limitations under the License.
         flex-direction: column;
         justify-content: center;
 
-        .mx_RoomTile_title,
         .mx_RoomTile_subtitle {
-            width: 100%;
+            align-items: center;
+            color: $secondary-content;
+            display: flex;
+            gap: $spacing-4;
+            line-height: $font-18px;
+        }
 
-            /* Ellipsize any text overflow */
-            text-overflow: ellipsis;
+        .mx_RoomTile_subtitle_text {
```

Long titles don't properly overflow with an ellipsis anymore, instead they break into the next line. The title also needs to be prevented from wrapping or overflowing, as it was before.

```suggestion
        .mx_RoomTile_title,
        .mx_RoomTile_subtitle_text {
```